doc-content/enterprise-only/vehicle-route-planning/backend-architecture-con.adoc

```diff
@@ -23,10 +23,11 @@ org.optaweb.vehiclerouting
 │   ├── persistence
 │   ├── planner
 │   ├── routing
-│   └── websocket
+│   └── rest
```

Comment:
@emmurphy1 Yup, this (and the `error` package) is the only necessary change. The flat structure from Anna's screenshot is just a different way her IDE displays the packages.